### PR TITLE
Order Page

### PR DESCRIPTION
**Pull request by jmichelotti**
Elements on the order page - no logic for checkout and api call for creating order

### CODE REVIEW

File: ux/src/components/DistributionInformation/DistributionInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                Delivery will be an extra $5 charge. We delivery within 20 miles!
+            </Alert>
+            {isDelivery &&
+                <div className="justify-content-center">
+                    <div className="form-group">
+                        <label className="control-label">Full Name</label>
+                        <input type="text" className="form-control" id="full_name_id" name="full_name" placeholder="John Deer" />
+                    </div>
+
+                    <div className="form-group">
```
Comment (smichelotti) [inline ux/src/components/DistributionInformation/DistributionInformation.tsx:29]: we'll probably need to change this to the <Forms> stuff taht comes with react-bootstrap - so we can get validation, etc. like I did in the admin site. But for now, let's leave as is.

File: ux/src/components/MenuItemDisplay/MenuItemDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+export const MenuItemDisplay: React.FunctionComponent<MenuItemDisplayProps> = (props) => {
+    return (
+        <>
+            <div className="container" style={{ padding: '10px' }}>
+                <div className="menu-item">
+                    <div className="row">
+                        <div className="col-md-3">
+                            <img src={props.item.imageUrl} alt="logo" style={{ padding: '15px' }} />
```
Comment (smichelotti) [inline ux/src/components/MenuItemDisplay/MenuItemDisplay.tsx:15]: consider using a css class for this - looks like you're using it in multiple places

File: ux/src/pages/Order/Order.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                {!loading && <h4>This is the order now page</h4>}
+                <Accordion defaultActiveKey="0">
+                    <Card>
+                        <Card.Header>Menu</Card.Header>
+                        <Accordion.Collapse eventKey="0">
+                            <Card.Body>
+                                <div>
+                                    {data.map((item, i) => {
+                                        return (
+                                            <div key={i}>
```
Comment (smichelotti) [inline ux/src/pages/Order/Order.tsx:27]: not sure if this <div> is necessary

**Review by smichelotti — APPROVED** [review]

File: ux/src/pages/Order/Order.tsx (hunk 3 above)
Comment (jmichelotti) [reply ux/src/pages/Order/Order.tsx:27]: Which part

Comment (smichelotti) [reply ux/src/pages/Order/Order.tsx:27]: the github markdown hid what i typed - let me try again :)  I don't think this `<div>` tag on line 27 is need

Comment (jmichelotti) [reply ux/src/pages/Order/Order.tsx:27]: yup I just took it out and it still works
